mlflow/transformers/__init__.py

```diff
@@ -2669,7 +2733,7 @@ def decode_audio(encoded):
     @staticmethod
     def _validate_str_input_uri_or_file(input_str):
         """
-        Validation of blob references to audio files, if a string is input to the ``predict``
+        Validation of blob references to audio/image files, if a string is input to the ``predict``
```

Comment:
```suggestion
        Validation of blob references to either audio or image files, if a string is input to the ``predict``
```
Let's not use shorthand in docstrings